conn/pool.go

```diff
@@ -63,10 +66,12 @@ func init() {
 	pi.all = make(map[string]*Pool)
 }
 
+// Get returns the list of pools.
```

Comment:
This is a bad name, `conn.Get` returns all the pools of connections?
It'd be better `conn.Pools` returns the list of pools.

Reply:
Pools is already taken by the type name but I changed it to GetPools